README.md

```diff
@@ -41,6 +41,23 @@ $ scalingo run "gm version"
 
 ## Hacking
 
+### New way
+
+You could upload the desired version of GraphicMagick from [GraphicMagick sources](https://hg.osdn.net/view/graphicsmagick/GM/tags)
+on a S3 for example. Then using the environment variable `ARCHIVE_URL` to specify
+the archive location, following the example:
+
+```shell
+scalingo env-set ARCHIVE_URL=https://hg.osdn.net/view/graphicsmagick/GM/archive/GraphicsMagick-1_3_36.tar.gz
+```
+
+> Note that Scalingo provide only support of the default version defined in `configs.sh` file.
+We do not guarantee the proper functioning of the build in case of usage of other versions.
+
```

Comment:
Maybe add some docs for the `GRAPHICS_MAGICK_VERSION` env var?

Reply:
I don't think it is useful, because if a user wants to use its own archive URL, he doesn't need to set this variable.
The use of this variable will be mostly internal, at the end.